lib/web_ui/test/engine/pointer_binding_test.dart

```diff
@@ -3087,6 +3087,385 @@ void testMain() {
       packets.clear();
     },
   );
+
+  group('$ClickDebouncer', () {
```

Comment:
```suggestion
  group('ClickDebouncer', () {
```

We don't do the `$` anymore (at least on the ecosystem :P)

Reply:
Done.